src/main.ts

```diff
@@ -13,6 +13,13 @@ import { LoggingInterceptor } from '~/common/interceptors/logging.interceptor';
 import { WebhookResponseCodeInterceptor } from '~/common/interceptors/webhook-response-code.interceptor';
 import { PolymeshLogger } from '~/logger/polymesh-logger.service';
 
+// This service was originally designed with node v14, this ensures a backwards compatible run time
+// Ideally we wouldn't need this function, but I am unable to find the cause when submitting SDK transactions that fail validation
+const unhandledRejectionLogger = new Logger('UnhandledPromise');
```

Comment:
I can't figure out why the service crashes without this. This makes the service work like before though without breaking when called with the wrong node flag.